### DISCUSSION TITLE
Serializing in browser

### DISCUSSION THREAD

**Discussion by joelmo** [activej/activej#179] [Q&A]
Do you have plans to use activej serialization in modern web browsers?

**Comment by eduard-vasinskyi** [comment]
Hello, @joelmo 
The ActiveJ Serializer is for the server-side. It is primarily used for logs, big data processing, and microservice architecture.

**Comment by EbiPenMan** [comment]
I also need the JavaScript version.
With the benchmark I got from it, I want to use this for communication between client and server in the WebSocket protocol.
